src/main/java/com/betleopard/domain/Horse.java

```diff
@@ -38,5 +39,27 @@ public String getName() {
     public String toString() {
         return "Horse{" + "name=" + name + ", id=" + id + '}';
     }
+
+    @Override
+    public int hashCode() {
+        int hash = 7;
+        hash = 59 * hash + Objects.hashCode(this.name);
```

Comment:
59 (111011) is a poor choice of multiplier due to five ones in its binary representation. A better choice is 73 (if you need a number larger than the standard 37).